### PR TITLE
Modularize geometry category

### PR DESCRIPTION
**Pull request by jenmerritt**
### Description

- modularizes the geometry category

Checklist:
- [ ] Set target branch to main (current release) or v.next (next release)
- [ ] Request a reviewer
- [ ] Assign a reviewer
- [ ] Tag reviewer in comment

### CODE REVIEW

**Comment by jenmerritt** [comment]
Ready for review @Luiy0

File: geometry/geodesic-operations/src/main/java/module-info.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+module com.esri.samples.geodesic_operations {
+  // require ArcGIS Runtime module
+  requires com.esri.arcgisruntime;
+
+  // require JavaFX modules that the application uses
+  requires javafx.graphics;
+  requires javafx.controls;
+
+  exports com.esri.samples.geodesic_operations;
+}
```
Comment (Luiy0) [inline geometry/geodesic-operations/src/main/java/module-info.java]: Newline here.

File: geometry/geodesic-sector-and-ellipse/src/main/java/module-info.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  // require JavaFX modules that the application uses
+  requires javafx.graphics;
+  requires javafx.controls;
+  requires javafx.fxml;
+
+  // make all @FXML annotated objects reflectively accessible to the javafx.fxml module
+  opens com.esri.samples.geodesic_sector_and_ellipse to javafx.fxml;
+
+  exports com.esri.samples.geodesic_sector_and_ellipse;
+}
```
Comment (Luiy0) [inline geometry/geodesic-sector-and-ellipse/src/main/java/module-info.java]: Newline here.

**Review by Luiy0 — APPROVED** [review]
LGTM

**Pull request merged**